### ISSUE TITLE
Option to remove imported files

### ISSUE THREAD

**Issue by mikkotikkanen** [jakubpawlowicz/clean-css#536]
If you are running master css file which is just split with imports, it would be nice to have r.js styled removing of the imported files

Ie. if

**master.css**

``` css
@import('buttons.css')
body { background: #fff; }
```

**buttons.css**

``` css
.btn { background: #bada55; }
```

Would then become...

**master.css**

``` css
.btn { background: #bada55; }
body { background: #fff; }
```

...with `buttons.css` removed.

If file is not imported anywhere, it would be just minified as usual.

**Comment by jakubpawlowicz** [comment]
@mikkotikkanen can you point me to relevant r.js functionality? I'd say it's rather a build system functionality rather than a minifier role to remove files.

**Comment by mikkotikkanen** [comment]
https://github.com/jrburke/r.js/blob/870ef5140c2fc89867db3f47541bbcf61f4cd38d/build/example.build.js#L349

Yeah, not really sure who's responsibility it is. I guess when running directly from the CLI, it should be the `clean-css`, but when using as part of the build I guess it should be fe. gulp, though it would be only `clean-css` which would know which files actually have been combined, not the next step in the build system. Just removing others than the `master.css` is not really an option neither since some files are included conditionally (like ie fixes etc).

**Comment by jakubpawlowicz** [comment]
I'm uneasy about this option at this point so let me think about it first.

Re gulp it's not a problem as clean-css can (and maybe should?) expose a list of inlined files via API.

**Comment by mikkotikkanen** [comment]
Yeah, at least the gulp option should be there. :)

And regardless, it should be bind separate options trigger (`removeCombined` or whatnot) anyhow.

**Comment by jakubpawlowicz** [comment]
Moved to https://github.com/jakubpawlowicz/clean-css-cli/issues/1

**Comment by jakubpawlowicz** [comment]
@mikkotikkanen it's been released as a part of clean-css-cli 4.1
